<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-FE2EINF" failures="1" tests="4" name="Test" time="22.419" errors="0" timestamp="2022-10-09T22:28:12 IST">
  <testcase classname="com.SauceDemo_TestClasses.TC01_VerifyLoginFunctionality" name="verifyLoginFunctionality" time="0.201"/>
  <testcase classname="com.SauceDemo_TestClasses.TC02_VerifyLogoutFunctionality" name="VerifyLogoutFunctionality" time="0.537"/>
  <testcase classname="com.SauceDemo_TestClasses.TC03_VerifyBagAddToCartFunctionality" name="VerifyBagAddToCartFunctionality" time="0.442"/>
  <testcase classname="com.SauceDemo_TestClasses.TC04_VerifyAllEleAddToCartFunctionality" name="VerifyAllEleAddToCartFunctionality" time="1.019">
    <failure type="java.lang.AssertionError" message="expected [6] but found [3]">
      <![CDATA[java.lang.AssertionError: expected [6] but found [3]
at com.SauceDemo_TestClasses.TC04_VerifyAllEleAddToCartFunctionality.VerifyAllEleAddToCartFunctionality(TC04_VerifyAllEleAddToCartFunctionality.java:34)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- VerifyAllEleAddToCartFunctionality -->
</testsuite> <!-- Test -->
